src/components/Form.tsx

```diff
@@ -4,63 +4,96 @@ import Button from "./Button";
 import Fieldset from "./Fieldset";
 
 export interface FormProps {
-  content?: Array<JSX.Element>;
+  /** An element or array of elements that will be rendered in the body of the <form>. */
+  content?: JSX.Element | Array<JSX.Element>;
+  /** The action to perform on the <form>'s onSubmit function and,
+  if there is a <button>, on the <button>'s onClick function. */
   onSubmit: any;
   title?: string;
+  className?: string;
   hiddenName?: string;
   hiddenValue?: string;
+
+  /** By default, the <button>'s content will be the string "Submit". */
   buttonContent?: string | JSX.Element;
   buttonClass?: string;
-  className?: string;
-  errorText?: string;
-  successText?: string;
-  infoText?: string;
   disableButton?: boolean;
+  /** Set this to true if you 1) want to use your own button element rather than the
+  reusable <Button> from this repo, or 2) want some other element (e.g. a checkbox) to
```

Comment:
I’d remove checkbox from this as an example since we mentioned a checkbox would not be ideal for this “other element”.